examples/with-viem/src/contracts.ts

```diff
@@ -0,0 +1,104 @@
+import * as path from "path";
+import * as dotenv from "dotenv";
+
+import { createAccount } from "@turnkey/viem";
+import { TurnkeyClient } from "@turnkey/http";
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+import {
+  createWalletClient,
+  createPublicClient,
+  http,
+  type Account,
+} from "viem";
+import { goerli } from "viem/chains";
+import { print } from "./util";
+import abi from "./abi.json";
+
+// Load environment variables from `.env.local`
+dotenv.config({ path: path.resolve(process.cwd(), ".env.local") });
+
+async function main() {
+  const turnkeyClient = new TurnkeyClient(
+    {
+      baseUrl: process.env.BASE_URL!,
+    },
+    new ApiKeyStamper({
+      apiPublicKey: process.env.API_PUBLIC_KEY!,
+      apiPrivateKey: process.env.API_PRIVATE_KEY!,
+    })
+  );
+
+  const turnkeyAccount = await createAccount({
+    client: turnkeyClient,
+    organizationId: process.env.ORGANIZATION_ID!,
+    privateKeyId: process.env.PRIVATE_KEY_ID!,
+  });
+
+  const client = createWalletClient({
+    account: turnkeyAccount as Account,
+    chain: goerli,
+    transport: http(
+      `https://goerli.infura.io/v3/${process.env.INFURA_API_KEY!}`
+    ),
+  });
+
+  const address = client.account.address;
+  print("Address:", address);
+
+  const account = client?.account?.address;
+  const hash = await client.writeContract({
+    abi,
+    address: "0x85111DF47D97Cc1aB3e76889fCF08594d053E85d",
+    functionName: "mint",
+    chain: goerli,
+    account,
```

Comment:
The call will succeed if `account` is omitted. However, if included, it leads to:

```
/Users/andrew/tkhq/code/sdk/node_modules/.pnpm/viem@1.10.0_typescript@5.1.3/node_modules/viem/dist/cjs/utils/errors/getTransactionError.js:11
    return new transaction_js_1.TransactionExecutionError(cause, {
           ^

TransactionExecutionError: The method does not exist / is not available.

URL: https://goerli.infura.io/v3/84842078b09946638c03157f83405213
Request body: {"method":"eth_sendTransaction","params":[{"data":"0x1249c58b","from":"0xd1FE65c7925f56594277bdb22FbFC4a2cB613514","to":"0x85111DF47D97Cc1aB3e76889fCF08594d053E85d"}]}
 
Request Arguments:
  chain:  Goerli (id: 5)
  from:   0xd1FE65c7925f56594277bdb22FbFC4a2cB613514
  to:     0x85111DF47D97Cc1aB3e76889fCF08594d053E85d
  data:   0x1249c58b

Details: The method eth_sendTransaction does not exist/is not available
Version: viem@1.10.0
    at getTransactionError (/Users/andrew/tkhq/code/sdk/node_modules/.pnpm/viem@1.10.0_typescript@5.1.3/node_modules/viem/dist/cjs/utils/errors/getTransactionError.js:11:12)
    at sendTransaction (/Users/andrew/tkhq/code/sdk/node_modules/.pnpm/viem@1.10.0_typescript@5.1.3/node_modules/viem/dist/cjs/actions/wallet/sendTransaction.js:77:44)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at writeContract (/Users/andrew/tkhq/code/sdk/node_modules/.pnpm/viem@1.10.0_typescript@5.1.3/node_modules/viem/dist/cjs/actions/wallet/writeContract.js:12:18)
    at main (/Users/andrew/tkhq/code/sdk/examples/with-viem/src/contracts.ts:49:16) {
  details: 'The method eth_sendTransaction does not exist/is not available',
  docsPath: undefined,
  metaMessages: [
    'URL: https://goerli.infura.io/v3/84842078b09946638c03157f83405213',
    'Request body: {"method":"eth_sendTransaction","params":[{"data":"0x1249c58b","from":"0xd1FE65c7925f56594277bdb22FbFC4a2cB613514","to":"0x85111DF47D97Cc1aB3e76889fCF08594d053E85d"}]}',
    ' ',
    'Request Arguments:',
    '  chain:  Goerli (id: 5)\n' +
      '  from:   0xd1FE65c7925f56594277bdb22FbFC4a2cB613514\n' +
      '  to:     0x85111DF47D97Cc1aB3e76889fCF08594d053E85d\n' +
      '  data:   0x1249c58b'
  ],
  shortMessage: 'The method does not exist / is not available.',
  version: 'viem@1.10.0',
  cause: /Users/andrew/tkhq/code/sdk/node_modules/.pnpm/viem@1.10.0_typescript@5.1.3/node_modules/viem/dist/cjs/utils/buildRequest.js:40
                      throw new rpc_js_1.MethodNotFoundRpcError(err);
                            ^
  
  MethodNotFoundRpcError: The method does not exist / is not available.
  
  URL: https://goerli.infura.io/v3/84842078b09946638c03157f83405213
  Request body: {"method":"eth_sendTransaction","params":[{"data":"0x1249c58b","from":"0xd1FE65c7925f56594277bdb22FbFC4a2cB613514","to":"0x85111DF47D97Cc1aB3e76889fCF08594d053E85d"}]}
  
  Details: The method eth_sendTransaction does not exist/is not available
  Version: viem@1.10.0
      at delay.count.count (/Users/andrew/tkhq/code/sdk/node_modules/.pnpm/viem@1.10.0_typescript@5.1.3/node_modules/viem/dist/cjs/utils/buildRequest.js:40:27)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      at attemptRetry (/Users/andrew/tkhq/code/sdk/node_modules/.pnpm/viem@1.10.0_typescript@5.1.3/node_modules/viem/dist/cjs/utils/promise/withRetry.js:15:30) {
    details: 'The method eth_sendTransaction does not exist/is not available',
    docsPath: undefined,
    metaMessages: [
      'URL: https://goerli.infura.io/v3/84842078b09946638c03157f83405213',
      'Request body: {"method":"eth_sendTransaction","params":[{"data":"0x1249c58b","from":"0xd1FE65c7925f56594277bdb22FbFC4a2cB613514","to":"0x85111DF47D97Cc1aB3e76889fCF08594d053E85d"}]}'
    ],
    shortMessage: 'The method does not exist / is not available.',
    version: 'viem@1.10.0',
    cause: /Users/andrew/tkhq/code/sdk/node_modules/.pnpm/viem@1.10.0_typescript@5.1.3/node_modules/viem/dist/cjs/clients/transports/http.js:40
                        throw new request_js_1.RpcRequestError({
                              ^
    
    RpcRequestError: RPC Request failed.
    
    URL: https://goerli.infura.io/v3/84842078b09946638c03157f83405213
    Request body: {"method":"eth_sendTransaction","params":[{"data":"0x1249c58b","from":"0xd1FE65c7925f56594277bdb22FbFC4a2cB613514","to":"0x85111DF47D97Cc1aB3e76889fCF08594d053E85d"}]}
    
    Details: The method eth_sendTransaction does not exist/is not available
    Version: viem@1.10.0
        at request (/Users/andrew/tkhq/code/sdk/node_modules/.pnpm/viem@1.10.0_typescript@5.1.3/node_modules/viem/dist/cjs/clients/transports/http.js:40:27)
        at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
        at delay.count.count (/Users/andrew/tkhq/code/sdk/node_modules/.pnpm/viem@1.10.0_typescript@5.1.3/node_modules/viem/dist/cjs/utils/buildRequest.js:30:20)
        at attemptRetry (/Users/andrew/tkhq/code/sdk/node_modules/.pnpm/viem@1.10.0_typescript@5.1.3/node_modules/viem/dist/cjs/utils/promise/withRetry.js:15:30) {
      details: 'The method eth_sendTransaction does not exist/is not available',
      docsPath: undefined,
      metaMessages: [Array],
      shortMessage: 'RPC Request failed.',
      version: 'viem@1.10.0',
      cause: [Object],
      code: -32601
    },
    code: -32601
  }
}
```